src/main/resources/archetype-resources/silo/src/main/resources/application-testing.yml

```diff
@@ -2,5 +2,9 @@ spring:
     main:
         show_banner: false
 
+    jpa:
+         hibernate:
+           ddlAuto: none
```

Comment:
Hm, you are right. It's not in the archetype, we just did it in the individual projects.